common/data_refinery_common/migrations/0038_paginate_default_svd_algorithm.py

```diff
@@ -1,26 +1,36 @@
 from django.db import migrations, models
-from django.core.paginator import Paginator
+
+def batch_update(queryset, batch=1000, **changes):
+    """ Update per batch """
+    is_done = False
+    current = 1
+
+    while not is_done:
+        start = (current - 1) * batch
+        end = start + batch
+        pks = queryset.values_list('pk', flat=True)[start:end][::1]
```

Comment:
What's `[::1]` do?

Reply:
same as list(queryset)

Reply:
thought it was easier to read but may have been mistaken